src/components/table/table.types.ts

```diff
@@ -23,6 +23,11 @@ export interface Column<T extends object = any> {
      * Type of aggregator to use for the column
      */
     aggregator?: ColumnAggregatorType | ColumnAggregatorFunction<T>;
+
+    /**
+     * Action menu icon used to display a menu component when clicked
```

Comment:
This is not 100% true, a consumer may choose what they want to do when the icon is clicked. Maybe we should rename this property to something else to indicate that :thinking: What do you think? Maybe `headerIcon` or something similar is better?